conformance/conformance.go

```diff
@@ -173,6 +188,7 @@ func RunConformance(t *testing.T) {
 // RunConformanceWithOptions runs the Inference Extension conformance tests with specific options.
 func RunConformanceWithOptions(t *testing.T, opts confsuite.ConformanceOptions) {
 	t.Logf("Running Inference Extension conformance tests with GatewayClass %s", opts.GatewayClassName)
+	t.Logf("CONFORMANCE.GO RunConformanceWithOptions: BaseManifests path being used by opts: %q", opts.BaseManifests)
```

Comment:
if we have this, we do not need Line 137 IMO

Reply:
Done. Removed live 137. I was debugging an issue and got a bit carried away with logging :)